src/Stepper/Stepper.tsx

```diff
@@ -12,19 +12,18 @@ const propTypes = {
   style: PropTypes.object,
   steps: PropTypes.arrayOf(PropTypes.shape({
     title: PropTypes.string.isRequired,
-    description: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+    description: PropTypes.node,
     success: PropTypes.bool,
-    warning: PropTypes.bool,
+    warning: PropTypes.string,
     optional: PropTypes.bool,
   })).isRequired,
-  currentStep: PropTypes.number,
-  seekable: PropTypes.bool,
+  currentStep: PropTypes.string,
   stickySidebar: PropTypes.bool,
   onStepClick: PropTypes.func,
 };
 
 const defaultProps = {
-  currentStep: 0,
+  currentStep: "0",
```

Comment:
not going to work if a client is using semantic IDs - you can pick `steps[0].id` as the default in the `render` method if no `currentStepID` is specified.
better yet, just make `currentStepID` required, so things are clear and explicit. the client inevitably has to track step changes anyway, so we might as well require the initial to be specified to ensure we're always in sync with the client.